src/appleseed/foundation/meta/tests/test_objmeshfilereader.cpp

```diff
@@ -121,4 +139,41 @@ TEST_SUITE(Foundation_Mesh_OBJMeshFileReader)
         EXPECT_EQ(4, mesh.m_tex_coords.size());
         EXPECT_EQ(1, mesh.m_faces.size());
     }
+
+#if 0
```

Comment:
Why do we keep this ?

Reply:
This code is useful when we need to convert an image to code data. I want to keep it around and not have to rewrite it every time or fish it from an outdated branch. I don't think it really hurts to keep it here.